src/stac_utils/google.py

```diff
@@ -20,10 +20,7 @@ def get_credentials(
     """Loads a Google Service Account into a Credentials object with the given scopes"""
 
     if not service_account_blob:
-        try:
-            service_account_blob = json.loads(os.environ[service_account_env_name])
-        except (json.JSONDecodeError, KeyError):
-            pass
+        service_account_blob = json.loads(os.environ[service_account_env_name])
```

Comment:
Will be easier to debug this if we catch the error, log it, and throw a useful error message instead of letting the JSON error propagate up.